x = -5
if x < 0:
  raise valueError('x should be postive')

#raise is a manuall way to throw the exception 
